ivy/functional/frontends/numpy/ndarray/ndarray.py

```diff
@@ -703,3 +600,110 @@ def __rshift__(self, value, /):
 
     def __lshift__(self, value, /):
         return ivy.bitwise_left_shift(self.ivy_array, value)
+
+
+# Helpers #
+# ------- #
+
+
```

Comment:
Could you please have a look through all files and remove old headers like here? It's a bit of a chore but needs to be done, otherwise we'll have duplicates.